fake/fake_clock.go

```diff
@@ -406,6 +443,51 @@ func (f *Clock) AwaitTimerAborts(n int) {
 	}
 }
 
+// AwaitAggExtractedChans waits the aggregate number of calls to Ch() on
+// timers to equal or exceed its argument.
+// To be be most useful, uses of the channel should directly call `.Ch()` on
+// the timers and dereferencing the channel pointer.
```

Comment:
This seems a bit unclear to me. Maybe more like:

For this method to be most useful, users of timers should not store the value of `.Ch()`. Instead, call `.Ch()`, dereference the pointer, and attempt a receive immediately, as in `case <-*timer.Ch()`.

Reply:
I like your wording 🙂 
Dropped in verbatim.